source/reference/minio-mc-admin/mc-admin-config.rst

```diff
@@ -1404,8 +1350,7 @@ using these environment variables.
          :start-after: start-minio-notify-nats-address
          :end-before: end-minio-notify-nats-address
 
-      This configuration setting corresponds with the environment variable
-      :envvar:`MINIO_NOTIFY_NATS_ADDRESS`.
+      This configuration setting corresponds with the environment variable :envvar:`MINIO_NOTIFY_NATS_ADDRESS`.
```

Comment:
The ones in this section are following a different sentence structure. I'd modify the ones in this section to follow the same format as all the others.

```suggestion
      This configuration setting corresponds with the :envvar:`MINIO_NOTIFY_NATS_ADDRESS`  environment variable.
```

Reply:
All of them down through line 1530.

Reply:
Ah, I think those were existing "ok-ish" ones that didn't match the fixes I was searching for. Good catch.